lib/elixir/lib/map.ex

```diff
@@ -1,6 +1,22 @@
 defmodule Map do
   @moduledoc """
   A set of functions for working with maps.
+  
+  Please note many functions for maps--which implement the `Enumerable` protocol
+  --can be found in `Enum`.
+  
+  Additionally, the following functions for maps are found in `Kernel`:
+  
+    * `get_and_update_in/2`
+    * `get_and_update_in/3`
+    * `get_in/2`
+    * `map_size/1`
+    * `pop_in/1`
+    * `pop_in/2`
+    * `put_in/2`
+    * `put_in/3`
+    * `update_in/2`
+    * `update_in/3`
```

Comment:
Mh, I am not sure I agree with listing all the `Access` calls that are in `Kernel` here. Those work on more than maps (for example, keyword lists).